README.md

```diff
@@ -1 +1,162 @@
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+**[Hello Microverse Project]** is the first project we build at Microverse. It just a setup of everything usefule for a web development.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Setting up Linters properly]**
+- **[Html and css]**
+- **[Mobile view]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- - [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up, follow these steps:
+- Open your Terminal
+- Navigate to the desired folder
+-git clone https://github.com/rbd3/Hello-Microverse-Project.git
+
+### Prerequisites
+
+In order to run this project you need:
+
+you need a browser like google chrome
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+git clone https://github.com/rbd3/Hello-Microverse-Project.git
+```
+### Install
+
+This project doesn't need installation.
+
+### Usage
+
+To run the project, execute the following command:
+
+double click on the index.html file
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+Hello-Microverse-Project/test.md
+```
+
+
+### Deployment
+
+You don't need a deployment from now.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@rbd3](https://github.com/rbd3)
+- Twitter: [@Narson321](https://twitter.com/@Narson321)
+- LinkedIn: [@Andry Narson Rabedesana](https://linkedin.com/in/andry-narson-rabedesana-15b8b4248)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[more project i finish]**
+- [ ] **[good view of the website]**
+- [ ] **[some interaction with visitor]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give this repository a 5 star rating.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank:
+- Microverse Review Team
+- My Coding partener
+- my family too for supporting me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Please add a License file (MIT is highly recommended) and also add the link to the file on `line 161` to enable others to freely use, change, and distribute the project in your repository 😄